src/test/java/net/masterthought/cucumber/json/EmbeddingWithNameTest.java

```diff
@@ -34,24 +27,16 @@ public static Iterable<Object[]> data() {
             { "text/php", "echo 'Hello World!';", "hello-world.php7", NO_DECODING, ".php7" },
         });
     }
-
-    @Parameter(0)
     public String mimeType;
-
-    @Parameter(1)
     public String data;
-
-    @Parameter(2)
     public String name;
-
-    @Parameter(3)
     public String decodedData;
-
-    @Parameter(4)
     public String fileName;
 
-    @Test
-    public void getMimeType_ReturnsMimeType() {
+    @MethodSource("data")
+    @ParameterizedTest(name = "\"{0}\" into \"{2}\"")
```

Comment:
@hazendaz  after this change tests are still skipped - can you kindly remind me whether was it intentional or tests should be executed and passing again?

Reply:
Will send PR.  Its the 2 tests that have assumptions to them.  When those assumptions are encountered those tests are skipped.  There are 2 methods like that so total of 12 tests skipped.

Reply:
Also went ahead and sent some other pull requests I had sitting still.  Nothing critical but I do have some probably 30 commits total I still had not pushed up otherwise so I took a few and pushed up just a while ago.

Reply:
Great, will merge and then perform release!